api/models/Chat.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+
+  attributes: {
+    username : {
+      type: 'string',
+      required: false
+    },
+    message: {
+      type: 'string',
+      required: true
+    },
+    type: {
+      type: 'string',
+      required: true
+    },
+    time: {
+      type: 'integer',
```

Comment:
I know you prefer ints for dates, but should this be a datetime? Moment can do all the heavy lifting of tinezones for us.